pkgs/development/libraries/libkrunfw/default.nix

```diff
@@ -44,6 +51,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/containers/libkrunfw";
     license = with licenses; [ lgpl2Only lgpl21Only ];
     maintainers = with maintainers; [ nickcao ];
-    platforms = [ "x86_64-linux" "aarch64-linux" ];
+    broken = stdenv.isDarwin && stdenv.isx86_64;
```

Comment:
So this only works on `aarch64-darwin`?

Reply:
Yes @ https://github.com/containers/krunvm/blob/main/README.md:


Supported platforms:
Linux/KVM on x86_64.
Linux/KVM on AArch64.
macOS/Hypervisor.framework on ARM64.

Reply:
Then you can just add aarch64-darwin to platforms, it looks cleaner this way.

Reply:
Done.